WatchFaceFormat/Weather/watchface/src/main/res/raw/watchface.xml

```diff
@@ -75,114 +134,29 @@
-                    </Compare>
-                    <Compare expression="is_thunderstorm">
-                      <PartImage height="47" width="47" x="0" y="0">
-                        <Image resource="ic_thunderstorm"/>
-                      </PartImage>
-                    </Compare>
-                    <Compare expression="is_sleet">
-                      <PartImage height="47" width="47" x="0" y="0">
-                        <Image resource="ic_sleet"/>
-                      </PartImage>
-                    </Compare>
-                    <Compare expression="is_mist">
-                      <PartImage height="47" width="47" x="0" y="0">
-                        <Image resource="ic_mist"/>
-                      </PartImage>
-                    </Compare>
-                    <Compare expression="is_partly_cloudy_day">
-                      <PartImage height="47" width="47" x="0" y="0">
-                        <Image resource="ic_partly_cloudy_day"/>
-                      </PartImage>
-                    </Compare>
-                    <Compare expression="is_partly_cloudy_night">
-                      <PartImage height="47" width="47" x="0" y="0">
-                        <Image resource="ic_partly_cloudy_night"/>
-                      </PartImage>
-                    </Compare>
-                    <Compare expression="is_windy">
-                      <PartImage height="47" width="47" x="0" y="0">
-                        <Image resource="ic_windy"/>
-                      </PartImage>
-                    </Compare>
-                    <Compare expression="is_atmospheric_condition">
-                      <PartImage height="47" width="47" x="0" y="0">
-                        <Image resource="ic_atmospheric_condition"/>
-                      </PartImage>
-                    </Compare>
-                    <Compare expression="is_thunder">
-                      <PartImage height="47" width="47" x="0" y="0">
-                        <Image resource="ic_thunder"/>
-                      </PartImage>
-                    </Compare>
-                    <Compare expression="is_tornado">
-                      <PartImage height="47" width="47" x="0" y="0">
-                        <Image resource="ic_tornado"/>
-                      </PartImage>
-                    </Compare>
-                    <Compare expression="is_tropical_cyclone">
-                      <PartImage height="47" width="47" x="0" y="0">
-                        <Image resource="ic_tropical_cyclone"/>
-                      </PartImage>
+                    <Compare expression="is_day">
+                      <PartText name="current_condition_icon" height="47" width="47" x="0" y="0">
+                        <Text>
+                          <BitmapFont family="weather_icons_day" size="47">
+                            <Template><![CDATA[%s]]>
+                              <Parameter expression="[WEATHER.CONDITION]"/>
```

Comment:
`[WEATHER.CONDITION] &lt;= 15 : [WEATHER.CONDITION] : 0` in case the value is somehow out of the expected range?

Reply:
Added in https://github.com/android/wear-os-samples/pull/1132/commits/059ced6fb668edbb8b8ab38e9cb6957b3275c08c